src/cmd/build.js

```diff
@@ -32,8 +32,13 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let ext = 'zip';
+          // override file extension if appropriate
+          if (fileExtension) {
+            ext = fileExtension;
```

Comment:
hmm. I'm a little concerned that someone would accidentally add a dot which could make the filename `package..xpi`. Perhaps the `fileExtension` value should be stripped of all non `[a-zA-Z0-9_-]` characters?